src/time-picker/TimePicker.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { props, t, skinnable } from '../utils';
+import cx from 'classnames';
+import Dropdown from '../dropdown/Dropdown';
+import FlexView from '../flex/FlexView';
+import range from 'lodash/utility/range';
+import flatten from 'lodash/array/flatten';
+
+const H24 = '24h';
+const H12 = '12h';
+const symbolRegex = /[\s,\.\|\/\\]/;
+const numberRegex = /^\d+$/;
+const separator = ':';
+
+const pad = (num) => num <= 9 ? `0${num}` : num;
+const timeFormatter24 = (hour, minute) => `${pad(hour)}:${pad(minute)}`;
+const timeFormatter12 = (hour, minute) => {
+  if (hour === 0) {
+    return `12${separator}${pad(minute)} am`;
+  } else if (hour === 12) {
+    return `12${separator}${pad(minute)} pm`;
+  } else if (hour >= 13) {
+    return `${pad(hour % 12)}${separator}${pad(minute)} pm`;
+  } else {
+    return `${pad(hour)}${separator}${pad(minute)} am`;
+  }
+};
+
+const lteTime = (minTime, maxTime) => (
```

Comment:
yep @giogonzo propose `lte` because `compareTime` doesn't looks very clear, at actually i'm agree with this, maybe without `Time`? Is defo a `lte` function, we should find out a better name